metadata-ingestion/build.gradle

```diff
@@ -279,6 +279,8 @@ clean {
   delete '.mypy_cache'
   delete '.pytest_cache'
   delete '.preflight_sentinel'
+  delete '.coverage'
```

Comment:
This was not here before. Is it there on purpose?

Reply:
Yes during this process noticed this was not being deleted so added this.